cypress-tests/cypress/e2e/admin/pageBuilder/templates/pageTemplatesCrud.cy.ts

```diff
@@ -0,0 +1,49 @@
+context("Page Builder - Template CRUD", () => {
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllTemplates();
+    });
+
+    it("Should be able to create a template and then edit and delete it", () => {
+        cy.visit("/page-builder/page-templates");
+
+        // Creates a template using the UI.
+        cy.findAllByTestId("pb-templates-list-new-template-btn").eq(0).click();
+        cy.findByRole("textbox", { name: "Title" }).type("testingfunctionality");
+        cy.findByRole("textbox", { name: "Slug" }).type("testingfunctionality");
+        cy.findByRole("textbox", { name: "Description" }).type("testingfunctionality");
+        cy.findByRole("button", { name: "Create" }).click();
+        cy.findByRole("button", { name: "Save Changes" }).should("exist").click();
```

Comment:
Added an empty line and a comment.

```suggestion
        cy.findByRole("button", { name: "Create" }).click();
        
        // In the page editor, we click Save Changes and return back to list of templates.
        cy.findByRole("button", { name: "Save Changes" }).should("exist").click();
```